pkg/render/manager.go

```diff
@@ -447,6 +466,9 @@ func (c *managerComponent) managerEsProxyContainer() corev1.Container {
 		Image:           components.GetReference(components.ComponentEsProxy, c.installation.Spec.Registry, c.installation.Spec.ImagePath),
 		LivenessProbe:   c.managerEsProxyProbe(),
 		SecurityContext: securityContext(),
+		VolumeMounts: []corev1.VolumeMount{
```

Comment:
Same comment as above. (Do we need this all the time?)

Reply:
It can be removed.